src/clj/athens/self_hosted/web/datascript.clj

```diff
@@ -23,6 +23,8 @@
     :datascript/split-block-to-children
     :datascript/unindent
     :datascript/indent
+    :datascript/indent-multi
+    :datascript/unindent-multi})
```

Comment:
Can you check if these characters, `})`, need to be removed?

Reply:
yeah you are right this happened during merge resolution